src/js/components/molecules/tooltip-toggle/tooltip-toggle.ts

```diff
@@ -37,45 +40,44 @@ export class TooltipToggle extends Vue implements TooltipToggleProps {
         this.visible = true;
     }
 
-    toggle(): void {
+    showTooltip():void {
         if (!this.visible) {
             this.show();
         } else {
             this.hide();
         }
     }
 
-    hide():void {
+    hideTooltip(): void {
+        if (this.visible)  {
+            this.hide();
+        }
+    }
+
+    async hide():Promise<void> {
         this.visible = false;
     }
 
     tooltipEnter(el, done) {
         const tooltipElement = <HTMLElement> el;
-        const iconElement = <HTMLElement> this.$refs.button.$el;
-        const { height: tooltipHeight, width: tooltipWidth } = <DOMRect> tooltipElement.getBoundingClientRect();
-        const { x: iconX, width: iconWidth } = <DOMRect> iconElement.getBoundingClientRect();
+        const clickableELement = this.$refs.button;
 
-        const iconCenterOffset = iconWidth / 2;
-        let xAxisCorrection = tooltipWidth / -2 + iconCenterOffset;
+        // @ts-ignore
+        const { height: elementHeight } = <DOMRect> clickableELement.getBoundingClientRect();
 
-        if (iconX + xAxisCorrection < 0) {
-            xAxisCorrection = 0;
-            this.tooltipLeft = true;
-        } else {
-            this.tooltipLeft = false;
-        }
+        this.yAxisCorrection =  elementHeight + (this.buildingUnit / 2);
```

Comment:
Edge detection is removed? the icon can be positioned near the left border of the screen on mobile devices. Without this code the tooltip can be positioned over the edge and become unreadable.